voluncheer/map/models.py

```diff
@@ -1,3 +1,19 @@
-# from django.db import models
+from django.db import models
+
 
 # Create your models here.
+class NYCharities(models.Model):
+    class Meta:
+        verbose_name_plural = "NYCharities"
+
+    name = models.CharField(max_length=255)
+    street = models.CharField(max_length=255)
+    city = models.CharField(max_length=255)
+    state = models.CharField(max_length=2)
+    type = models.CharField(max_length=255)
+    bin_num = models.PositiveBigIntegerField(default=0)
```

Comment:
I'm not sure I understand what the `bin_num` provides us that the ID of the object does not. Can you clarify it a bit? Also, what does the name mean?

Reply:
There is a field in the data named bin; there isn't a description for it, but I think it is like a bank number. There is no ID number for the NYCharities data set - that's why I am using the combination of the organization name, organization type, and bin number as a way to check if a record in the data set already exists in our NYCharities model. If it does, then we can update the address if there are any changes.

Reply:
My concern here was that the address for an nycharity object would be updated if necessary since this is important for our map.

Reply:
I see. Can you provide a comment just above the `bim_num = ...` line explaining what it is then? When a field can't be easily identified (without prior knowledge) just by looking at the code, that's when comments come in handy. Better yet, if you can provide a link or reference to the source that would be ideal.

Reply:
By link/ reference do you mean a reference to the data set?

Reply:
I added the comments and the link to the data set.

Reply:
Let me know if it's what you had in mind

Reply:
I definitely committed it last night... and it shows the new comments on my branch. I'll commit again